pkg/controller/datadogagentdeployment/datadogagentdeployment_controller.go

```diff
@@ -53,7 +56,19 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileDatadogAgentDeployment{client: mgr.GetClient(), scheme: mgr.GetScheme(), recorder: mgr.GetEventRecorderFor("DatadogAgentDeployment")}
+	reconciler := &ReconcileDatadogAgentDeployment{
+		client:   mgr.GetClient(),
+		scheme:   mgr.GetScheme(),
+		recorder: mgr.GetEventRecorderFor("DatadogAgentDeployment"),
+		datadogMetricsForwarder: &datadog.MetricsForwarder{
+			RetryInterval:        10 * time.Second,
```

Comment:
Curious if it is worth exposing these settings via pflag config parameter?

Reply:
yes, I believe the operator configurability must be discussed at this point

Reply:
regarding `RetryInterval` it might even best if we implement an exponential backoff strategy, like retry after 10 sec, 20 sec, 30 sec etc ..